examples/Deployment/HelloWorld.h

```diff
@@ -37,7 +37,7 @@ extern "C"
 typedef struct HelloWorld
 {
     uint32_t index;
-    char message[255];
+    char message[100];
```

Comment:
Max default size of a string is 255.

Reply:
My fault, it must be corrected at MicroRTPSGen